fdmi/ut/filterc_ut.c

```diff
@@ -154,11 +150,10 @@ static int ut_filterc_fco_start(struct m0_filterc_ctx *ctx,
 
 static void rpc_client_and_server_start(void)
 {
-	int rc;
-
-	rc = m0_net_domain_init(&client_net_dom, m0_fdmi_ut_xprt);
+	int                 rc;
```

Comment:
No need to change the variable alignment.

Reply:
incorporated